account_move_line_purchase_info/models/account_move.py

```diff
@@ -1,18 +0,0 @@
-# Copyright 2019 ForgeFlow S.L.
-#   (https://www.forgeflow.com)
-# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
-
-from odoo import fields, models
-
-
-class AccountMoveLine(models.Model):
-
-    _inherit = "account.move.line"
-
-    purchase_id = fields.Many2one(
```

Comment:
I think we should add a migration script to move this to the new standard field `purchase_order_id`.

Reply:
I update my comment: the field purchase_order_id is not stored, so therefore the filters added in this module won't work if we don't make it stored again.

Reply:
There are two options here:
- Move purchase_id field to the new standard field purchase_order_id, and make it stored: there is a risk on dependent fields. Dependent fields may not me computed correctly. A migration script will be needed to rename the fields.
- Keep separate fields: duplicated fields is a dirty option but safer and no migration script needed. But dirty

After choosing one option of those, can you please update the roadmap in the readme with the details of what and why we chose that option?

Reply:
@LoisRForgeFlow  @AaronHForgeFlow Thank you for your suggestions. I choose option

- > Move purchase_id field to the new standard field purchase_order_id, and make it stored: there is a risk on dependent fields. Dependent fields may not me computed correctly. A migration script will be needed to rename the fields.

Now I add a migration script, which you can review again.